org-brain.el

```diff
@@ -2355,6 +2361,10 @@ Setting NOFOCUS to t implies also having NOHISTORY as t.
 Unless WANDER is t, `org-brain-stop-wandering' will be run."
   (interactive
    (progn
+     (if (and (not (check-org-brain-exist-p))
```

Comment:
Here you could use `when` instead of `if`, since you don't have an else clause. That way you wouldn't need the `progn`. I also don't quite get the logic here. To me it seems like it will switch to the buffer it it doesn't exist?

Reply:
Thanks for the suggestion on how to properly use `when` and `if`, I
revised it in 37ba935f0ff3e8b2dff097c9d9623f796b0f284c and drop
`progn` in c6e851ac7b8bc7dbc038116c7c91979a998c68f6.

>To me it seems as it will switch to the buffer it doesn't exist?

Yes. When there is no **org-brain** buffer, just open a new **org-brain**
buffer. However, when there is an *org-brain* buffer, just switch to
it. The content is what was left.